server/opts.go

```diff
@@ -756,6 +758,10 @@ func parseCluster(v interface{}, opts *Options, errors *[]error, warnings *[]err
 				*errors = append(*errors, err)
 				continue
 			}
+			if val, exists := FlagSnapshot.inCmdLine["Cluster.TLSInsecure"]; exists {
```

Comment:
No, the FlagSnapshot is for config reload. You should not be using it here. So remove this `if` statement.

Reply:
Sorry, but i have no idea how you want me to do that and pass proper flags into getTLSConfig.